packages/core/src/browser/saveable.ts

```diff
@@ -179,6 +185,14 @@ export namespace SaveableWidget {
     }
 }
 
+export interface SaveOptions {
+    readonly skipFormattingOnSave?: boolean;
```

Comment:
I believe it comes from https://github.com/eclipse-theia/theia/pull/8543, but I think the naming can be simplified to:

```ts
/**
* Controls whether formatting should be applied upon saving.
*/
readonly skipFormatting? boolean;
```

It is clear that these are `saving` options by the naming of the `SaveOptions` interface.

Reply:
Thank you for your feedback. I've updated #8543